sdk/storage/azure-storage-blob/samples/blob_samples_authentication.py

```diff
@@ -106,3 +107,24 @@ def auth_shared_access_signature(self):
             expiry=datetime.utcnow() + timedelta(hours=1)
         )
         # [END create_sas_token]
+
+    def auth_default_azure_credential(self):
+        # [START create_blob_service_client_oauth]
+        # Get a credential for authentication
+        # Default Azure Credentials attempt a chained set of authentication methods, per documentation here: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity
+        # For example user (who must be an Azure Event Hubs Data Owner role) to be logged in can be specified by the environment variable AZURE_USERNAME
```

Comment:
> (who must be an Azure Event Hubs Data Owner role)

I suspect that this is just a typo - since this content is focused on blob storage and not event hubs?